scipy/ndimage/morphology.py

```diff
@@ -1736,6 +1736,24 @@ def white_tophat(input, size=None, footprint=None, structure=None,
     output : ndarray
         Result of the filter of `input` with `structure`.
 
+    Examples
+    --------
+    Subtract gray background from a bright peak.
+
+    >>> from scipy.ndimage import generate_binary_structure, white_tophat
+    >>> square = generate_binary_structure(rank=2, connectivity=3)
+    >>> bright_on_gray = numpy.array([[2, 3, 3, 3, 2],
```

Comment:
just a minor/superficial thing from my side--the CI is complaining because we use `np` directly in our docstrings instead of `numpy` (same for other docstring example below)